src/alexapi/device_platforms/raspberrypi.py

```diff
@@ -64,7 +64,7 @@ def indicate_recording(self, state=True):
 	def indicate_playback(self, state=True):
 		GPIO.output(self.__pconfig['plb_light'], GPIO.HIGH if state else GPIO.LOW)
 
-	def detect_button(self, channel):
```

Comment:
Then it is better to add that method with this signature to abstract class. And a comment about this in code, just in case somebody will wonder why it is there.